docs/concepts/further-reading/academic-papers.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Academic papers
+issueUrl: https://github.com/ipfs/docs/issues/479
+---
+
+# Academic papers
+
+Here are a few papers that are useful for understanding IPFS, whether it be understanding the IPFS spec itself, or the background for the decentralized web, protocols, hashing, and so on.
+
+## [A practicable approach towards secure key-based routing](https://web.archive.org/web/20170809130252id_/http://www.tm.uka.de/doc/SKademlia_2007.pdf)
+
+**Baumgart, Ingmar and Mies, Sebastian**: _Security is a common problem in completely decentralized peer-to-peer systems. Although several suggestions ex-ist on how to create a secure key-based routing protocol, a practicable approach is still unattended. In this paper we introduce a secure key-based routing protocol based on Kademlia that has a high resilience against common attacks by using parallel lookups over multiple disjoint paths, limiting free nodeId generation with crypto puzzles and intro-ducing a reliable sibling broadcast. The latter is needed tostore data in a safe replicated way. We evaluate the securityof our proposed extensions to the Kademlia protocol ana-lytically and simulate the effects of multiple disjoint pathson lookup success under the influence of adversarial nodes_
+
+## [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/papers/raw/master/ipfs-cap2pfs/ipfs-p2p-file-system.pdf)
```

Comment:
I would put the IPFS paper first on the list of further reading about IPFS :)

Reply:
Good call.